src/main/java/Task.java

```diff
@@ -0,0 +1,19 @@
+// Add the ability to store whatever text entered by the user and display them back to the user when requested.
+public class Task {
+    
+
+    String[] taskList = new String[100];
+    int taskCount = 0;
+
+    public void addTask(String description){
```

Comment:
Space needed before `{`


```suggestion
    public void addTask(String description) {
```